index.php

```diff
@@ -1,22 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+</head>
+<body>
+    
 <?php
+require __DIR__ . '/vendor/autoload.php';
 
-$curl = curl_init();
+$client = new GuzzleHttp\Client;
 
-// curl_setopt_array($curl, array(
-//     CURLOPT_URL => "https://ir-dev-d9.innoraft-sites.com/jsonapi/node/services" ,
-//     CURLOPT_HTTPHEADER => array(
-//         "Content-Type: text/plain"
-//     ),
-// ));
+for($i=15;$i>=0;$i--){
```

Comment:
This is still static, how do you know that data will remain 15 here?

Reply:
changed